src/Latte/Macros/BlockMacros.php

```diff
@@ -71,6 +74,7 @@ public function finalize()
 				$func = '_lb' . substr(md5($this->getCompiler()->getTemplateId() . $name), 0, 10) . '_' . preg_replace('#[^a-z0-9_]#i', '_', $name);
 				$snippet = $name[0] === '_';
 				$prolog[] = "//\n// block $name\n//\n"
+					. "\$_b->types[" . var_export($name, TRUE) . "][] = " . var_export($this->blockTypes[$name], TRUE) . ";"
```

Comment:
Ok, it is probably needless. I will update it and will try do find some failing test :)